examples/nlp/masked_language_modeling.py

```diff
@@ -301,27 +304,142 @@ def bert_module(query, key, value, i):
 
 
 class MaskedLanguageModel(keras.Model):
-    def train_step(self, inputs):
+
+    def train_step(self, *args, **kwargs):
```

Comment:
Looking at the code for the custom train step below, I don't think a custom train step is needed at all, I think it can be removed.

The only things that might need to be overridden are `compute_loss` and `compute_metrics`, and those can be implemented in a backend agnostic way. This would greatly simplify the code.

My guess is that this example was created before `compute_loss` existed.